server/src/main/java/io/deephaven/server/session/SessionService.java

```diff
@@ -240,6 +243,7 @@ public SessionState getSessionForAuthToken(final String token) throws Authentica
         final String payload = offset < 0 ? "" : token.substring(offset + 1);
         AuthenticationRequestHandler handler = authRequestHandlers.get(key);
         if (handler == null) {
+            log.info().append("No AuthenticationRequestHandler registered for type ").append(key).endl();
```

Comment:
revert this as well

Reply:
Actually this one is okay, since we throw otherwise

Reply:
No, changed my mind again, going to revert this until we do more testing - it turns out that we permit Bearer to be an auth type outside of what DH uses for refresh, but only if we can't handle the token _as a_ refresh token, so clients that were connected before restart can trigger this.